.github/workflows/release.yaml

```diff
@@ -202,3 +202,11 @@ jobs:
           aws s3 cp ./packages/account/src/providers/assets/images/ s3://${S3_CDN_BUCKET}/assets/ --recursive
         env:
           S3_CDN_BUCKET: ${{ secrets.S3_CDN_BUCKET }}
+
+      - name: Notify migrations and disclosures of the new release (breaking changes)
```

Comment:
Will this pull the new docs as well?

Reply:
Looks like it'll only update the `migrations-and-disclosures` [repository](https://github.com/FuelLabs/migrations-and-disclosures/blob/f6123525f6c58f01e4717bda1ecd7ca9d70eab45/.github/workflows/migrations.yml#L9-L10) with our version.

I assume then the process of updating the `docs-hub` is then manual from there.